apps/pie-storybook/stories/pie-toggle-switch.stories.ts

```diff
@@ -1,22 +1,34 @@
 import { html, TemplateResult } from 'lit';
 import type { StoryObj as Story } from '@storybook/web-components';
-import { PieToggleSwitch, ToggleSwitchProps } from '@justeattakeaway/pie-toggle-switch';
+import '@justeattakeaway/pie-toggle-switch';
```

Comment:
```suggestion
import { PieToggleSwitch } from '@justeattakeaway/pie-toggle-switch';
```

You should be able to directly import the component and add it to the kept references array. I'm not 100% sure the best approach but I'm using modal as the most up to date reference and this keeps us consistent.

Reply:
@kevinrodrigues let's please revert this to match the generator output and then look into this in a separate PR if importing this way would make more sense. 🥇

Reply:
Fixed.